CMSgov/design-system
# [WNMGDS-1250] Added `DrawerManager` context provider and `useDrawerManager` hook to manage state of multiple drawers on a single page.

## Summary

- [RFC](https://github.com/CMSgov/design-system/discussions/1951)
- Adds a `<DrawerManager>` context provider and `useDrawerManager()` hook for managing drawer state so that multiple drawers don't overlap one another.
- Added documentation with example
- Added unit test
- Updated VRT snapshots

## How to test

1. `yarn clean && yarn install && yarn build`
2. documentation update: `yarn start` then -> http://localhost:8000/components/drawer/#managing-multiple-drawers
3. `yarn test:unit`
4. `yarn storybook` then see drawer manager example

## Checklist


### If this is a change to code:
